panel/__init__.py

```diff
@@ -58,6 +58,7 @@
 from . import widgets  # noqa
 from .config import __version__, config, panel_extension as extension  # noqa
 from .depends import bind, depends  # noqa
+from .esm import JSComponent, PreactComponent, ReactComponent  # noqa
```

Comment:
Are we sure `esm` is the best name? We could also call it `base`, `custom`, `component` or `any` as `esm` is not very familiar in Python space.

Reply:
I like custom.

Reply:
Closing. Its a duplicate.